modules/end-user-guide/partials/ref_parameters-for-che-editor-yaml.adoc

```diff
@@ -28,23 +28,34 @@ reference: https://__<hostname_and_path_to_a_remote_file>__/che-editor.yaml
 inline:
   schemaVersion: 2.1.0
   metadata:
-    name: theia-ide
-  commands:
-    ...
-  events:
-    ...
+    name: IntelliJ IDEA Community Edition
   components:
-    - name: che-machine-exec
+    - name: intellij
       container:
-        image: 'quay.io/eclipse/che-machine-exec:next'
-        command:
-          ...
-        memoryLimit: 128Mi
+        image: 'quay.io/che-incubator/che-idea@sha256:b02663b2aea521bcbcafdbec7eba27b6d804e3885dc54bb7d5c771ed77d4a97f'
```

Comment:
why pick a specific sha? you could use :next` here...

Reply:
Apologies for that. I just plagiarized a file from Artem's test repo.
Can I rewrite it like this?
```Suggestion
```suggestion
        image: 'quay.io/che-incubator/che-idea:next'
```